### ISSUE TITLE
Changelog missing

### ISSUE THREAD

**Issue by joepie91** [gulpjs/gulp-util#111]
There does not appear to be a changelog anywhere, for this module. This makes it really hard to determine what has changed across releases, and whether it is safe to update to a new major version (in my particular case, from 2.2.17 to 3.0.7).

**Comment by contra** [comment]
This module is deprecated, and nothing will be changed in the future so there isn't really a point in starting now.

**Comment by joepie91** [comment]
Hmm, I'm not seeing a deprecation notice anywhere. What is it being replaced with?

**Comment by terinjokes** [comment]
@joepie91 all the functionality of this module is being deprecated in favor of requiring the functionality separately. Most of this module is already thin wrappers over other modules, or other modules simply re-exported.

**Comment by joepie91** [comment]
Okay... is there a migration guide of any kind?

**Comment by contra** [comment]
@joepie91 Let me rephrase: gulp-util doesn't and has never really done anything, it just exports other modules. Migration guide? Use the modules that gulp-util was giving you on their own.

**Comment by phated** [comment]
We are probably going to log deprecation warnings with suggested modules to switch to.  Locking this thread.
